documentation/docs/getting-started/configuration.mdx

```diff
@@ -118,24 +117,25 @@ Please use this table as a reference.
 | OPAL_POLICY_REPO_SSH_KEY                      | The content of the var is a private crypto key (i.e: SSH key). You will need to register the matching public key with your repo. For example, see the{" "} GitHub tutorial {" "} on the subject. The passed value must be the contents of the SSH key in one line (replace new-line with underscore, i.e: \n with{" "} \_). |                                      |
 | OPAL_POLICY_REPO_CLONE_PATH                   | Where (i.e: base target path) to clone the repo in your docker filesystem (not important unless you mount a docker volume).                                                                                                                                                                                                 |                                      |
 | OPAL_POLICY_REPO_MAIN_BRANCH                  | Name of the git branch to track for policy files (default: `master`).                                                                                                                                                                                                                                                       |                                      |
-| OPAL_BUNDLE_IGNORE                            | Comma separated list of glob paths to omit from policy bundle. Note that double asterisks \*\* do not recursively match.                                                                                                                                                                                                    | `bundle_ignore: Optional[List[str]]` |
+| OPAL_BUNDLE_IGNORE                            | Comma separated list of glob paths to omit from policy bundle. Note that double asterisks \*\* do not recursively match, unless at the end, and without other wildcards.                                                                                                                                                    | `bundle_ignore: Optional[List[str]]` |
```

Comment:
I liked the phrasing in `POLICY_STORE_POLICY_PATHS_TO_IGNORE` better: 
> or parent paths (without wildcards) ending with "/\*\*

It's good that you've reused the same capabilities for both features.